src/libexpr/primops.cc

```diff
@@ -49,6 +49,9 @@ StringMap EvalState::realiseContext(const PathSet & context)
             throw InvalidPathError(store->printStorePath(ctx));
         if (!outputName.empty() && ctx.isDerivation()) {
             drvs.push_back({ctx, {outputName}});
+            if (evalSettings.logAllIFD) {
+              printInfo("%1% importing from derivation %2% via %3%", pos, ctxS, reason);
```

Comment:
```suggestion
                printInfo("'import from derivation of '%s' by '%s' at %s", ctxS, reason, pos);
```
or something like that.